python/lsst/cell_coadds/_fits.py

```diff
@@ -136,6 +137,7 @@ def readAsMultipleCellCoadd(self) -> MultipleCellCoadd:
     def _readSingleCellCoadd(
         data: Mapping[str, Any],
         common: CommonComponents,
+        header: Mapping[str, Any],  # TODO: Use a more specific type here.
```

Comment:
Shouldn't "TODOs" have a ticket associated with them, with more details?
And there's another one above,L90: `units=CoaddUnits(1), # TODO: read from FITS TUNIT1`

Reply:
Because this is still in `lsst-dm` organization and not in `lsst` organization, the rules are relaxed. As long as these TODOs are addressed or assigned ticket numbers before porting, that should be good.

Reply:
Good point, I hadn't realized it was `lsst-dm`